rest/rest-server/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/SchedulerStateRest.java

```diff
@@ -1432,6 +1432,60 @@ private String getTaskResultValueAsStringOrExceptionStackTrace(TaskResult taskRe
     }
 
     /**
+     * Returns the metadata of the task result of task <code>taskName</code> of the
+     * job <code>jobId</code>.
+     *
+     * Metadata is a map containing additional information associated with a result. For example a file name if the result represents a file.
+     *
+     * @param sessionId
+     *            a valid session id
+     * @param jobId
+     *            the id of the job
+     * @param taskname
+     *            the name of the task
+     * @return the metadata of the task result
+     */
+    @GET
+    @GZIP
+    @Path("jobs/{jobid}/tasks/{taskname}/result/metadata")
+    @Produces("*/*")
+    public Map<String, String> metadataOfTaskResult(@HeaderParam("sessionid") String sessionId, @PathParam("jobid") String jobId,
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS1161)